include/ruby/internal/intern/string.h

```diff
@@ -601,6 +601,19 @@ VALUE rb_str_dup(VALUE str);
  */
 VALUE rb_str_resurrect(VALUE str);
 
+/**
+ * Returns whether a string is chilled or not.
+ *
+ * This function is temporary and users must check for its presence using
+ * mkmf's have_func helper. If this function is not implemented, then strings
+ * can't be chilled.
+ *
+ * @param[in]  str  A string.
+ * @retval     1    The string is chilled.
+ * @retval     0    Otherwise.
+ */
+bool rb_str_chilled_p(VALUE str);
+
```

Comment:
For convenience.
```suggestion

#define HAVE_RB_STR_CHILLED_P 1

```

Reply:
I was thinking the same, but then thought probably very few extensions need this so then it seems OK to use `have_func` for those extensions (`have_func` being rather slow).
If it's added the documentation above should be updated to mention no need to use `have_func`.

Reply:
I updated the PR with @nobu's suggestion.